src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -0,0 +1,126 @@
+                                label={translate('sulu_admin.enclosure')}
+                            >
+                                <SingleSelect onChange={this.handleEnclosureChanged} value={this.enclosure}>
+                                    <SingleSelect.Option value={'"'}>&quot;</SingleSelect.Option>
+                                    <SingleSelect.Option value="">
+                                        {translate('sulu_admin.enclosure_nothing')}
+                                    </SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                        </Form.Section>
+                        <Form.Section colSpan={6}>
+                            <Form.Field
+                                description={translate('sulu_admin.escape_description')}
+                                label={translate('sulu_admin.escape')}
+                            >
+                                <SingleSelect onChange={this.handleEscapeChanged} value={this.escape}>
+                                    <SingleSelect.Option value={'\\'}>\</SingleSelect.Option>
+                                    <SingleSelect.Option value={'"'}>&quot;</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                            <Form.Field
+                                description={translate('sulu_admin.new_line_description')}
+                                label={translate('sulu_admin.new_line')}
+                            >
+                                <SingleSelect onChange={this.handleNewLineChanged} value={this.newLine}>
+                                    <SingleSelect.Option value={'\\n'}>\n</SingleSelect.Option>
+                                    <SingleSelect.Option value={'\\r\\n'}>\r\n</SingleSelect.Option>
+                                    <SingleSelect.Option value={'\\r'}>\r</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                        </Form.Section>
+                    </Form>
+                </div>
+            </Overlay>
+        );
+    }
+
+    getToolbarItemConfig() {
+        return {
+            icon: 'su-download',
+            label: translate('sulu_admin.export'),
+            onClick: action(() => {
+                this.showOverlay = true;
+            }),
+            type: 'button',
+        };
+    }
+
+    @action handleClose = () => {
+        this.showOverlay = false;
+    };
+
+    @action handleDelimiterChanged = (value: string) => {
+        this.delimiter = value;
+    };
+    @action handleEnclosureChanged = (value: string) => {
+        this.enclosure = value;
+    };
+    @action handleEscapeChanged = (value: string) => {
+        this.escape = value;
+    };
+    @action handleNewLineChanged = (value: string) => {
```

Comment:
All of these function should have a line "padding" between them :slightly_smiling_face: